meinberlin/apps/dashboard2/views.py

```diff
@@ -173,8 +173,18 @@ def publish_project(self):
                              'Required fields are missing.'))
             return
 
+        responses = signals.project_pre_publish.send(sender=None,
+                                                     project=self.project)
```

Comment:
According to django documentation `sender` must be set:

>  You must provide the sender argument (which is a class most of the time) ...

Reply:
in the Signal source it states a class or None, so we are fine.